packages/sdk/src/connections/WindowConnection.ts

```diff
@@ -129,6 +129,29 @@ export class WindowConnection extends BaseConnection {
     window.postMessage(message, origin || window.origin);
   }
 
+  bindFuelConnectors(fuel: Window['fuel']) {
+    // Prevent binding to self if this happen the
+    // object would enter on a infinite loop
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const isSelf = (fuel as any) === this;
+    if (!fuel || isSelf) return;
+    // Bind to fuel events to
+    // sync with current instace
+    fuel.on('connectors', (connectors) => {
+      this.connectors = connectors;
+      this.emit('connectors', connectors);
```

Comment:
what is the point of emitting the event here?  Isn't the event already emitted by the fact that we are handling it here?

Reply:
The emit happens on the current instance. But the listener listens to the events on the `window.fuel` instance.